stream.go

```diff
@@ -100,7 +100,7 @@ type hasIdentity interface {
 
 type sendStreamMethods interface {
```

Comment:
Why did you change these to uint16 from ErrorCode?

Reply:
Because minq.ErrorCode is an enum that minq owns.  This is supposed to be a 16-bit slot for applications to do with as they please.  kQuicErrorNoError might appear to be a valid value for ErrorCode, but it is misleading because that codepoint is assigned to STOPPING.  All other codes are application-defined.